examples/native/coin/coin_contract.py

```diff
@@ -1,84 +1,123 @@
-import hydrachain.native_contracts as nc
-from ethereum import utils
+import ethereum.utils as utils
+from hydrachain import native_contracts as nc
 import ethereum.slogging as slogging
-log = slogging.get_logger('nc.coin')
+from nc_utils import STATUS, FORBIDDEN, NOTFOUND, OK, ERROR, BADREQEST, INSUFFICIENTFUNDS
+from nc_utils import isaddress
+log = slogging.get_logger('contracts.coin')
 
 
-DEBUG = utils.DEBUG
-
-STATUS = 'uint16'
-FORBIDDEN = 403
-NOTFOUND = 404
-OK = 200
-ERROR = 500
-BADREQEST = 400
-INSUFFICIENTFUNDS = PAYMENTREQUIRED = 402
+class Transfer(nc.ABIEvent):
 
+    "Triggered when tokens are transferred."
```

Comment:
Please use proper docstrings!
https://github.com/amontalenti/elements-of-python-style#writing-good-docstrings
```python
def get(url, qsargs=None, timeout=5.0):
    """Send an HTTP GET request.

    :param url: URL for the new request.
    :type url: str
    :param qsargs: Converted to query string arguments.
    :type qsargs: dict
    :param timeout: In seconds.
    :rtype: mymodule.Response
    """
return request('get', url, qsargs=qsargs, timeout=timeout)
```
Remember to use triple quotes.